gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/oc_service_manager_client.py

```diff
@@ -0,0 +1,123 @@
+# One Convergence, Inc. CONFIDENTIAL
```

Comment:
Actually the Service controller is going to run on an infra VM, this client library is something that the GBP driver on OS controller is going to use to talk to the Service controller. This is only the client library for GBP.